source/export-query-to-language.txt

```diff
@@ -8,6 +8,11 @@ Export Query to Specific Language
 
 *New in version 1.15.0*
 
+.. admonition:: Available in |compass| Only
```

Comment:
I like this note, I just think the title of the note and the content of the note are kind of confusing / overkill.

The title of the note says: "Available in MongoDB Compass Only" and the content says "...unavailable in the MongoDB Compass Community..." - I think that could confuse people (we're pushing users in two ways, at once).

Could we change the title of the note to just say "Note" (or something else) and then have the content say "This feature is unavailable in MongoDB Compass Community Edition.", or "This feature is unavailable in MongoDB Compass Community Edition. It is only available in MongoDB Compass."

Should we link to the Compass index page in the note?

Reply:
I really like the new note. The last comment I'll leave on this is whether the note should be yellow or red. The green makes it seems like I'm reading a useful pro tip or success message. Just my .02. Feel free to do what you think makes most sense here!

Reply:
Changed.